client/jetpack-cloud/sections/partner-portal/primary/payment-method-list/index.tsx

```diff
@@ -12,13 +12,24 @@ import Main from 'calypso/components/main';
 import CardHeading from 'calypso/components/card-heading';
 import DocumentHead from 'calypso/components/data/document-head';
 import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
-import { Button } from '@automattic/components';
+import { getAllStoredCards } from 'calypso/state/stored-cards/selectors';
+import QueryStoredCards from 'calypso/components/data/query-stored-cards';
+import AddStoredCreditCard from 'calypso/components/jetpack/add-stored-credit-card';
+import StoredCreditCard from 'calypso/components/jetpack/stored-credit-card';
```

Comment:
My bad here, I said `components` folder, but I wanted to say outside of the `primary` folder. We currently have the Partner Portal related components under `client/jetpack-cloud/sections/partner-portal/`.